client/blocks/product-selector/README.md

```diff
@@ -85,6 +85,7 @@ The following props can be passed to the Product Selector block:
 		}
 		```
 	* `optionsLabel`: ( string ) Title of the product options section.
+	* `productUpsells`: ( object ) Optional product upsells. Used to create a map of suggested higher-tier products as an up-sell for a purchased product.
```

Comment:
We could add here an example of the `productUpsells` structure, similar to what we have for e.g. the `productPriceMatrix`.

Reply:
Good idea, updated in https://github.com/Automattic/wp-calypso/pull/37250/commits/167ca5fb165d2223f09f880e14eca9a3ae803dfe